.travis.yml

```diff
@@ -6,10 +6,13 @@ rvm:
   - 2.1
   - 2.2
   - 2.3.0
+  - 2.4.0
 script:
   - bundle exec rake test
   - bundle exec rake coveralls:push
```

Comment:
We should probably remove coveralls. We haven't really been paying attention to it, and it causes errors when run internally.